workflow/signal/signal.go

```diff
@@ -45,6 +45,13 @@ func SignalContainer(restConfig *rest.Config, pod *corev1.Pod, container string,
 
 func ExecPodContainerAndGetOutput(restConfig *rest.Config, namespace string, pod string, container string, command ...string) error {
 	x, err := common.ExecPodContainer(restConfig, namespace, pod, container, true, true, command...)
+	log.
```

Comment:
do you want to move logging into `ExecPodContainer`?

Reply:
ok~, PTAL, thx